src-frontend/app/controllers/procedure.js

```diff
@@ -44,17 +57,20 @@ export default Ember.Controller.extend({
         updateSortOrder: function(pageModels) {
             var controller = this;
 
+            pageModels = pageModels || controller.generateNewPageIndices()
```

Comment:
I don't mind this convention since it's done on bash too. And yes, add the semicolon (how did this pass the linter?)

Is `pageModels` a global or local var? If it's global we should have a convention for them such as `m` prefix